programs/uxd/tests/integration_tests/api/program_uxd/instructions/mod.rs

```diff
@@ -8,6 +8,7 @@ pub mod process_initialize_identity_depository;
 pub mod process_mint_with_credix_lp_depository;
 pub mod process_mint_with_identity_depository;
 pub mod process_mint_with_mercurial_vault_depository;
```

Comment:
IF process everywhere, process nowhere prob fine too :p
for another day refactor hehe